business/workloads.go

```diff
@@ -391,7 +391,21 @@ func (in *WorkloadService) getParsedLogs(namespace, name string, opts *LogOption
 		if opts.IsProxy {
 			engardeParser := parser.New(parser.IstioProxyAccessLogsPattern)
 			al, err := engardeParser.Parse(entry.Message)
-			if err == nil {
+			// engardeParser.Parse will not throw errors even if no fields
```

Comment:
Sort of annoying behavior from Engarde.  Just out of curiosity, does al.ParseError get set or is any way useful?